src/main/scala/com/databricks/spark/redshift/RedshiftRelation.scala

```diff
@@ -38,14 +38,11 @@ case class RedshiftRelation(jdbcWrapper: JDBCWrapper, params: MergedParameters,
   with InsertableRelation
   with Logging {
 
-  override def schema = {
-    userSchema match {
-      case Some(schema) => schema
-      case None => {
-        jdbcWrapper.registerDriver(params.jdbcDriver)
-        jdbcWrapper.resolveTable(params.jdbcUrl, params.table, new Properties())
-      }
-    }
+  override def schema = userSchema match {
```

Comment:
If you're going to reformat this anyways then you might as well replace this with a `.getOrElse`.